### ISSUE TITLE
✨ Cleaner more understandable commit messages ✏️ 

### ISSUE THREAD

**Issue by danBamikiya** [benawad/dogehouse#1277]
**Is your feature request related to a problem? Please describe.**
Hi everyone, I'll like to suggest the use of cleaner, properly organised commit messages that everyone including new contributors can understand using [commitizen](https://www.npmjs.com/package/commitizen).

**Describe the solution you'd like**
I know you love your contribution workflows so this will be configured so you can keep commiting like before just with better organised commit messages using `commit-msg` hook with husky.

**Describe alternatives you've considered**

Like the below images but with `git commit` instead of `git cz` if that's possible with this repo.

[commitizen](https://www.npmjs.com/package/commitizen) can also be configured with [commitlint](https://github.com/conventional-changelog/commitlint#readme)

**Additional context**
![images](https://user-images.githubusercontent.com/58262528/111820513-e2dbb200-88e1-11eb-8b8b-952a8fea0733.jpg)
![67ce67daf5962ea50bc40eff969d7faf](https://user-images.githubusercontent.com/58262528/111820532-ea02c000-88e1-11eb-9ba2-6bf743687324.jpg)

<h2>If you're down for it, I can get to work. Its quite a simple feature.</h2>

**Comment by benawad** [comment]
commitzen is already setup just needs to be enforced, I will start doing that after new design hits

**Comment by danBamikiya** [comment]
OK that's nice. About scaling infrastructure have you considered Terraform? I can help with that! We could set up Terraform as Infrastructure and use Ansible for better configuration! 🏗️

**Comment by HarrisonMayotte** [comment]
I can also help with infrastructure as-needed.

**Comment by HarrisonMayotte** [comment]
Didn't mean to close this.

**Comment by danBamikiya** [comment]
I guess we have to wait for @benawad. From what I've observed its dockerized already and its using cloudflare as its provider. A very basic setup I have in mind is something like the below image.

![image](https://user-images.githubusercontent.com/58262528/111886774-a2a72d00-89d0-11eb-8513-52714355ab45.png)

**Comment by danBamikiya** [comment]
Will be scaled of course but still waiting on @benawad or any of the reviewers to collaborate

**Comment by HarrisonMayotte** [comment]
That seems like a good idea to me, but it comes back to Ben to make that decision.

**Comment by HarrisonMayotte** [comment]
I'm going to split this off to a separate issue as we aren't on commit messages anymore.

**Comment by ajmnz** [comment]
@benawad Time to enforce commitzen?

**Comment by Hypnotist1148** [comment]
Hey, you could use conventional commits.
Check it out [here](https://www.conventionalcommits.org/en/v1.0.0/)

ps: i like this project a lot :D

**Comment by goldyydev** [comment]
It's alive! (and enforced)
